website/docs/registries/package-tarball.md

```diff
@@ -42,14 +42,35 @@ The normalization process consists of the following:
    ```
 
 3. All sections other than `[package]`, `[dependencies]` and `[tool]` are removed from the manifest.
-4. All auto-detected properties, like `package.readme` are explicitly stated.
+4. All auto-detected properties, like `package.readme`, are explicitly stated.
 
 ### `Scarb.orig.toml`
 
 The original `Scarb.toml` file from the package source directory, without any processing.
 
+### `README` and `LICENSE`
+
+If these files are present in the project root, or if paths to them are provided in the manifest, they are copied to the archive as `README.md` and `LICENSE` respectively.
+
+### `VCS.json`
+
+File containing info about the version control system used in your package.
+Currently, only Git is supported.
```

Comment:
It's unclear why it's relevant to include in the tarball at all

Reply:
Can you please clarify whether it's `VCS.json` itself that concerns you or its presence in the docs?

Reply:
It concerns me that i don't know why the tool includes it in the tarball, what is its' purpose after packaging?

Reply:
I can't tell you for sure as I wasn't adding these [changes](https://github.com/software-mansion/scarb/pull/827), only documenting the current state of the tool. But, if I were to assume, this mirrors cargo, and will probably come in handy later on when Github-based package is verification introduced.

Not sure we need user to know why this exists, only that this is a reserved file name.